android/src/main/java/com/visioncameracodescanner/CodeScannerProcessorPlugin.java

```diff
@@ -31,8 +32,7 @@ public class CodeScannerProcessorPlugin extends FrameProcessorPlugin {
   private BarcodeScanner barcodeScanner = null;
   private int barcodeScannerFormatsBitmap = -1;
 
-  CodeScannerProcessorPlugin(@Nullable Map<String, Object> options) {
-    super(options);
```

Comment:
Why removing `super(options);`?

Reply:
The parent class, `FrameProcessorPlugin` [has dropped the `options` param in the latest version](https://github.com/mrousavy/react-native-vision-camera/blame/b1fa06514f4f2450ca2d0387d2ec781963ed4ef8/package/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorPlugin.java#L27).

As such, based on my (admittedly dated and possibly flawed) recollection of how Java constructors work, the implicit call to `super()` should be sufficient here.